QMediathekView.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>QMediathekView</id>
```

Comment:
I think this should also adjust `QMediathekView.pro` to install this file to `/usr/share/metainfo/%{id}.appdata.xml`?

Reply:
That would be great, yes.